client/src/pages/ManageBookingPage/BlankComponent.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: 360,
+    height: 125,
+    marginTop: theme.spacing(1),
+  },
+  contents: {
+    padding: theme.spacing(0.5),
+  },
+  name: {
+    float: 'left',
+    marginLeft: theme.spacing(1),
+    padding: theme.spacing(1.5),
+  },
+}));
+
+export default function BookingComponent(props) {
```

Comment:
This is supposed to be `BlankComponent` I think

Reply:
Yeah ;)